x-pack/plugins/maps/public/connected_components/map/mb/view.js

```diff
@@ -112,6 +113,7 @@ export class MBMapContainer extends React.Component {
   }
 
   async _createMbMapInstance() {
+    const initialView = await getInitialView(this.props.goto, this.props.settings);
```

Comment:
see comment above, this might end up being null, even though goto is set. While it depends a little on the timing of this merging, it's a safety.

Reply:
See above comment